src/test/run-pass-fulldeps/auxiliary/roman_numerals.rs

```diff
@@ -29,7 +29,7 @@ use rustc_plugin::Registry;
 // WARNING WARNING WARNING WARNING WARNING
 // =======================================
 //
-// This code also appears in src/doc/guide-plugin.md. Please keep
+// This code also appears in src/doc/unstable-book. Please keep
```

Comment:
Why not use the full path `src/doc/unstable-book/src/language-features/plugin.md`?

Reply:
I made it consistent with the manner in other code [feature_gate.rs#L106](https://github.com/rust-lang/rust/blob/c16f480fa6eb6b28f51c7218347bcaf1377dedea/src/libsyntax/feature_gate.rs#L106).
~~~
// If you change this, please modify src/doc/unstable-book as well.
~~~

Reply:
The difference is that the comment in `feature_gate.rs` applies to every feature listed in the code below it, so pointing to the folder makes sense. The test case here only refers to a single feature.

Reply:
Oh, I understand. Thank you @kennytm . I updated it.